Backend/Remora.Discord.Gateway/ResponderDispatchOptions.cs

```diff
@@ -0,0 +1,29 @@
+//
+//  ResponderDispatchOptions.cs
+//
+//  Author:
+//       Jarl Gullberg <jarl.gullberg@gmail.com>
+//
+//  Copyright (c) 2017 Jarl Gullberg
+//
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU Lesser General Public License for more details.
+//
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+
```

Comment:
Add using to stop the warnings here?

Reply:
Instead of a `using` only for the documentation `see` tag.
cref should point to the FullName with Namespace.

Reply:
I'm fine with adding a `using` just for this - it's how it's done elsewhere in the library.

Reply:
```suggestion
using Remora.Discord.Gateway.Services;

```